### PR TITLE
Keb operation storage

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- implement storage for operations insert/update/get in Kyma Environment Broker

**Related issue(s)**
Related: https://github.com/kyma-incubator/compass/issues/815

### CODE REVIEW

File: components/kyma-environment-broker/internal/storage/dbsession/write.go [hunk 1]
```diff
@@ -7,13 +7,46 @@ import (
...
+const (
+	// OperationTypeProvision means provisioning OperationType
+	OperationTypeProvision OperationType = "provision"
+	// OperationTypeDeprovision means deprovision OperationType
+	OperationTypeDeprovision OperationType = "deprovision"
+	// OperationTypeUndefined means undefined OperationType
+	OperationTypeUndefined OperationType = ""
+)
+
+type OperationDTO struct {
```
Comment (jasiu001) [inline components/kyma-environment-broker/internal/storage/dbsession/write.go]: Why this struct is here? not with others in the `internal` (model.go) package? `internal.Operation` cannot be used here?

File: components/schema-migrator/migrations/kyma-environment-broker/202002121000_add_operations.up.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS operations (
+			id varchar(255) PRIMARY KEY,
```
Comment (jasiu001) [inline components/schema-migrator/migrations/kyma-environment-broker/202002121000_add_operations.up.sql]: spaaaaace :)

File: components/kyma-environment-broker/internal/model.go [hunk 3]
```diff
@@ -22,6 +24,41 @@ type Instance struct {
...
 }
 
+type Operation struct {
+	ID        string
+	Version   int
+	CreatedAt time.Time
+	UpdatedAt time.Time
+
+	InstanceID        string
+	TargetOperationID string
```
Comment (jasiu001) [inline components/kyma-environment-broker/internal/model.go]: be more specific with this field name or add a comment

File: components/kyma-environment-broker/internal/model.go [hunk 4]
```diff
@@ -22,6 +24,41 @@ type Instance struct {
...
+	State             domain.LastOperationState
+	Description       string
+}
+
+// ProvisioningOperation holds all information about provisioning operation
+type ProvisioningOperation struct {
+	Operation `json:"-"`
+
+	// following fields are serialized to JSON and stored in the storage
+	LoggingTenantID        string `json:"logging_tenant_id"`
```
Comment (jasiu001) [inline components/kyma-environment-broker/internal/model.go]: maybe the name should tell about connection with mls

File: components/kyma-environment-broker/internal/storage/dbsession/write.go (hunk 1 above)
Comment (piotrmiskiewicz) [reply components/kyma-environment-broker/internal/storage/dbsession/write.go]: It is about how to store in the DB, it needs Type and Data

**Pull request merged**